### DISCUSSION TITLE
How to donate money for iOS version development

### DISCUSSION THREAD

**Discussion by drdavidliao** [jamulussoftware/jamulus#2650] [Ideas]
Dear Group (sorry if this is in the wrong place, admonition/redirection appreciated),

Is there a mechanism for donating money specifically for the development of the iOS (and Android) version(s) of Jamulus?

I would like to be able to tell people "just install Jamulus from the App Store by following this link." 

Jesse Chappell got SonoBus working on iOS (on App Store) and Android and helped Alex Carôt get a beta version of Soundjack running on iOS (distributed on TestFlight). I don't know what Jesse's availability is like, but if there were a bounty/fund I could donate to to help hire Jesse or someone like him to help bring the iOS version of Jamulus across the finish line, I would love to do that. 

Thank you,
David Liao

**Comment by pljones** [comment]
We need people to do the development.  No one gets paid.

**Reply by drdavidliao** [reply]
Thank you.

**Comment by ann0see** [comment]
Jamulus is somewhat available on the App Store but not recommended. You can also talk to @emlynmac who develops a Jamulus compatible app.
"Donations" are usually helping out with hosting, development, maintenance, testing, ... currently Jamulus is not too much money  consuming.

**Comment by emlynmac** [comment]
@LiaoSiQuENC I started writing something specific for iOS/macOS as a client-only, written in swift.
I've got it to a beta stage for iOS and have much of the implementation available for open source use.

The protocol implementation can be found here: 
https://github.com/emlynmac/JamlusProtocolSwift

The main audio engine can be found here:
https://github.com/emlynmac/JamulusAudioEngineSwift

I have a modified swift-opus library to support jamulus' custom modes here:
https://github.com/emlynmac/swift-opus
